core/math/src/main/java/org/arakhne/afc/math/geometry/d2/Transform2D.java

```diff
@@ -227,15 +227,15 @@ public void getTranslationVector(Tuple2D<?> translation) {
 	public void rotate(double theta) {
 		// Copied from AWT API
 		final double sin = Math.sin(theta);
-		if (sin == 1.) {
+		if (MathUtil.areFloatsEqual(sin, 1.)) {
```

Comment:
This causes invalid border effect. Additionally, it is not compliant with the Oracle code from which this function was copied.